compiler/ballerina-lang/src/test/resources/ballerina-toml/other-entries.toml

```diff
@@ -0,0 +1,10 @@
+firstField = "yee"
+
+[package]
+org = "foo"
+name = "winery"
+version = "1.0.0"
+newField = "fee"
+
+[userTable]
+name = "bar"
```

Comment:
Can we test with some numbers and also some complex tables as well

Reply:
Added